src/sensor/Sensor.cpp

```diff
@@ -0,0 +1,64 @@
+#include "sensor.h"
+#include <Arduino.h>
+
+// Définition des pins
+const int trigPin = 9;
+const int echoPin = 10;
+const int buttonPin = 7;  // Nouveau : bouton connecté à la pin 7
```

Comment:
nomenclature for variables and constants should be using underscores and not maj:

trig_pin,
echo_pin,
....